main/fopen_wrappers.c

```diff
@@ -110,6 +110,11 @@ PHPAPI ZEND_INI_MH(OnUpdateBaseDir)
 			*end = '\0';
 			end++;
 		}
+		if (ptr[0] == '.' && ptr[1] == '.' && ptr[2] == '\0') {
```

Comment:
I think we should also forbid anything starting with `../`, not just literally `..`.

Reply:
Hmm, yes, that should catch all issues.